go-controller/pkg/node/gateway_init_linux_test.go

```diff
@@ -201,6 +201,10 @@ func shareGatewayInterfaceTest(app *cli.App, testNS ns.NetNS,
 		existingNode := v1.Node{
 			ObjectMeta: metav1.ObjectMeta{
 				Name: nodeName,
+				Annotations: map[string]string{
```

Comment:
```
E0726 10:31:01.964013  514109 node_ip_handler_linux.go:307] Unable to parse addresses from node host node1: k8s.ovn.org/host-addresses annotation not found for node "node1"
I0726 10:31:01.964025  514109 node_ip_handler_linux.go:544] Node address changed. Updating annotations.
E0726 10:31:01.964239  514109 node_ip_handler_linux.go:547] Address Manager failed to update node address annotations: k8s.ovn.org/l3-gateway-config annotation not found for node "node1"
```
Test then fails because expected commands were not seen:
```
Timed out after 5.001s.
    Executed commands do not match expected commands!
    [00]   /fake-bin/ovs-vsctl --timeout=15 port-to-br eth0
    [01]   /fake-bin/ovs-vsctl --timeout=15 port-to-br eth0
    [02]   /fake-bin/ovs-vsctl --timeout=15 br-exists eth0
    [03]   /fake-bin/ovs-vsctl --timeout=15 -- --may-exist add-br breth0 -- br-set-external-id breth0 bridge-id breth0 -- br-set-external-id breth0 bridge-uplink eth0 -- set bridge breth0 fail-mode=standalone other_config:hwaddr=8a:47:6a:ad:3e:f3 -- --may-exist add-port breth0 eth0 -- set port eth0 other-config:transient=true
    [04]   /fake-bin/sysctl -w net.ipv4.conf.breth0.forwarding=1
    [05]   /fake-bin/ovs-vsctl --timeout=15 --if-exists get interface breth0 mac_in_use
    [06]   /fake-bin/ovs-vsctl --timeout=15 set bridge breth0 other-config:hwaddr=8a:47:6a:ad:3e:f3
    [07]   /fake-bin/ovs-vsctl --timeout=15 --if-exists get Open_vSwitch . external_ids:ovn-bridge-mappings
    [08]   /fake-bin/ovs-vsctl --timeout=15 set Open_vSwitch . external_ids:ovn-bridge-mappings=physnet:breth0
    [09]   /fake-bin/ovs-vsctl --timeout=15 --if-exists get Open_vSwitch . external_ids:system-id
    [10]   /fake-bin/ovs-appctl --timeout=15 dpif/show-dp-features breth0
    [11]   /fake-bin/ovs-vsctl --timeout=15 --if-exists get Open_vSwitch . other_config:hw-offload
    [12]   /fake-bin/ovs-vsctl --timeout=15 get Interface patch-breth0_node1-to-br-int ofport
    [13]   /fake-bin/ovs-vsctl --timeout=15 get interface eth0 ofport
    [14] ! /fake-bin/ovs-vsctl --timeout=15 get Open_vSwitch . external_ids:ovn-encap-ip
    [15] ! /fake-bin/ovs-vsctl --timeout=15 set Open_vSwitch . external_ids:ovn-encap-ip=192.168.1.10
    [16] ! /fake-bin/ovn-appctl --timeout=5 -t ovn-controller exit --restart
    [17]   /fake-bin/ip route replace table 7 172.16.1.0/24 via 10.1.1.1 dev ovn-k8s-mp0
    [18]   /fake-bin/ip -4 rule
    [19]   /fake-bin/ip -4 rule add fwmark 0x1745ec lookup 7 prio 30
    [20]   /fake-bin/sysctl -w net.ipv4.conf.ovn-k8s-mp0.rp_filter=2
    [21]   /fake-bin/ovs-ofctl -O OpenFlow13 --bundle replace-flows breth0 -
    [22]   /fake-bin/ovs-vsctl --timeout=15 --if-exists get interface patch-breth0_node1-to-br-int ofport
    [23]   /fake-bin/ovs-vsctl --timeout=15 --if-exists get interface eth0 ofport
    
    Expected
        <bool>: false
    to be true

    /home/mkennell/repos/ovn-kubernetes/go-controller/pkg/node/gateway_init_linux_test.go:336
------------------------------

```

Reply:
Understood this now and previously I did not - I removed the need for annotations.